### DISCUSSION TITLE
GraphQL error: FHIR request failed

### DISCUSSION THREAD

**Discussion by nsanzimanagilbert** [opencrvs/opencrvs-core#4157] [Technical Q&A] [answered]
Hi team!

Landed into this issue, it causes the application to load endlessly and finally throws that error in the console but with no error indicated in the logs or in the code itself. 
Any help over there?

![image](https://user-images.githubusercontent.com/38226948/198001841-54e99178-b850-4ab1-b6c7-00db31846010.png)

**Comment by nsanzimanagilbert** [comment]
![image](https://user-images.githubusercontent.com/38226948/198031396-d45d65d5-fb9e-4f63-8644-ce0e85164053.png)
After some good time, the logged error looks as shown above. However, the application remains in an infinite loop and never renders the pages.

**Comment by euanmillar** [comment, marked as answer]
Hey @nsanzimanagilbert from the first error, this usually happens if a mistake has been made in configuration.  It is usually related to issues encountered when setting up a default backup.  I suggest you run [these steps](https://documentation.opencrvs.org/setup/3.-installation/3.2-set-up-your-own-country-configuration) again.  Regarding the second error, it looks like your **workflow** service is not running as that is what uses port 5050.  Perhaps there is a code error there that you may have introduced or perhaps the port is in use by another application.  Try saving the [index](https://github.com/opencrvs/opencrvs-core/blob/develop/packages/workflow/src/index.ts) file in workflow while the application is running and see if you get any meaningful Node error information.  This is not replicable on our side.  This link can tell if if your stack is running: https://is-my-opencrvs-up.netlify.app/
